src/test/java/ru/itmo/sd/deadliner2bot/service/TaskFetchingServiceJpaTest.java

```diff
@@ -77,7 +77,21 @@ public void dailyNotificationOutOfRangeNotFetched() {
         createDailyNotification(chat, now.plusSeconds(2L * storedRangeSec));
 
         Set<DailyNotification> dailyNotifications = taskFetchingService
-                .getDailyNotifications(now.plusSeconds(storedRangeSec));
+                .getDailyNotifications(now, now.plusSeconds(storedRangeSec));
+        assertThat(dailyNotifications).isEmpty();
+        Set<TodoNotification> todoNotifications = taskFetchingService
+                .getTodoNotifications(now.plusSeconds(storedRangeSec));
+        assertThat(todoNotifications).isEmpty();
+    }
+
+    @Test
```

Comment:
I suggest to make test, which checks, that daily notifications are fetched correctly on next week (fistly, fetch with now time, after - fetch with now + 7 days time, should fetch the same daily notifications).
If you don't want to do it in this PR - you can skil.

Reply:
Done.